lib/rsyncwrapper.d.ts

```diff
@@ -21,8 +21,12 @@ interface RsyncOptions {
     args?: Array<string>;
 }
 
+interface RsyncError extends Error {
+    code: number;
```

Comment:
The error can be returned without a `code` property in case the entire try/catch block errors:

https://github.com/jedrichards/rsyncwrapper/blob/master/lib/rsyncwrapper.js#L194

Maybe should make the code property optional in that case?